build.gradle

```diff
@@ -36,6 +36,7 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-security'
     implementation 'com.auth0:auth0-spring-security-api:1.4.0'
     compile 'com.auth0:java-jwt:3.4.0'
+    compile group: 'org.springframework.boot', name: 'spring-boot-starter-validation', version: '2.3.7.RELEASE'
```

Comment:
what is the purpose of this?

Reply:
in the ProjectEntity this is used for the name @Size to be between 1 and 100